ui/app/adapters/job.js

```diff
@@ -49,9 +49,9 @@ export default Watchable.extend({
   xhrKey(url, method, options = {}) {
     const namespace = options.data && options.data.namespace;
     if (namespace) {
-      return `${url}?namespace=${namespace}`;
+      return `${method} ${url}?namespace=${namespace}`;
```

Comment:
Dunno why but I'm always wary of using%20spaces in keys.

Reply:
URL encoding haunts all our dreams.

To be honest, I want to rip all this stuff out and replace it with some form of cancellation token [like axios uses](https://github.com/axios/axios#cancellation).